src/partials/tutorial-info.hbs

```diff
@@ -19,8 +19,8 @@
   {{/with}}
   <div class="tutorial-actions">
     {{#with page.attributes.colab-link}}
-    <a href="{{{this}}}" target="_blank" class="colab btn btn-outlined btn-neutral">
-      <div class="mr-1">{{{svg 'colab'}}}</div>
```

Comment:
I realized the previous svg helper wasn't that useful since we can just use css to set this content.